pages/editpage/[id].js

```diff
@@ -1,5 +1,30 @@
-import CreateRecipe from "../../components/Form";
+import { useRouter } from "next/router";
+import EditRecipe from "../../components/Form/Edit";
 
-export default function CreatePage() {
-  return <CreateRecipe />;
+export default function EditRecipePage() {
+  const router = useRouter();
+  const { id } = router.query;
+
+  const handleSubmit = async (updatedRecipe) => {
+    try {
```

Comment:
I dont think you need a try/catch block here. You already have it in the API. If the fetch fails, you will fall into !response.ok

Reply:
I removed the try/catch block